examples/swift_protobuf_example/Package.swift

```diff
@@ -0,0 +1,15 @@
+// swift-tools-version: 6.0
+
+import PackageDescription
+
+let package = Package(
+    name: "swift_protobuf_example",
+    dependencies: [
+        // These are the versions used by rules_swift
+        // .package(url: "https://github.com/grpc/grpc-swift-extras", exact: "1.0.0-beta.2"),
+        // .package(url: "https://github.com/grpc/grpc-swift.git", exact: "2.0.0-beta.2"), // Required by grpc-swift-extras @ 1.0.0-beta.2
+        // .package(url: "https://github.com/grpc/grpc-swift-protobuf.git", exact: "1.0.0-beta.2"), // Required by grpc-swift @ 2.0.0-beta.2
+        // .package(url: "https://github.com/grpc/grpc-swift-nio-transport.git", exact: "1.0.0-beta.2"), // Required by grpc-swift @ 2.0.0-beta.2
```

Comment:
Do we need these comments? It mostly looks like commented code.

Reply:
No I can remove them